app/src/styles.sass

```diff
@@ -164,17 +163,52 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
   right: 0
   display: flex
   flex-flow: row nowrap
-  width: 80%
+  overflow: hidden
+  width: 80vw
+  height: 150vw
   @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-laptop)
-    width: 50%
+    width: 50vw
+    height: 70vw
   @media screen and (min-width: $breakpoint-laptop)
     display: none
-  .info-panel-icon #help-button
-    margin: 0.7em $small-space
+  .info-panel-content
+    position: absolute
+    z-index: 1
+    right: 0
+    width: 70vw
+    @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-laptop)
+      width: 45vw
+    @media screen and (min-width: $breakpoint-laptop)
+      display: block
   .info-panel-content .information
     display: contents
     p
       margin-top: 1em
+.info-panel.opened  
+  max-width: 80vw
+  min-width:32px
```

Comment:
Why do you need these two lines?

Reply:
I use these 2 lines to set the range of width when the info-panel is open

Reply:
.info-panel has `position: absolute` therefore it isn't in any other box. There is no constraint on its size and it always has the width set by the width property. Hence min-width and max-width are useless.

Reply:
yes the number of the width can be changes to be bigger or smaller. But I think on the web page, the width doesn't expend to be bigger than 80vw or shrink smaller than 32px.

Reply:
Yes, but the control of the width is here too. If someone in the future want to change the style such that the with of the open panel is 90vw, he will have two change two different properties (lines 167 and 188). Anyway, you can keep these two lines if you want.